internal/webserver/server.go

```diff
@@ -134,39 +174,33 @@ func (webserver *WebServer) listenAndServe(serviceTimeout time.Duration, errChan
 			errChannel <- err
 			return
 		}
-
 		httpsCert, ok := httpsSecretData[config.HttpServer.HTTPSCertName]
 		if !ok {
 			lc.Errorf("unable to find HTTPS Cert in Secret Data as %s. Check configuration", config.HttpServer.HTTPSCertName, err)
 			errChannel <- err
 			return
 		}
-
 		httpsKey, ok := httpsSecretData[config.HttpServer.HTTPSKeyName]
 		if !ok {
 			lc.Errorf("unable to find HTTPS Key in Secret Data as %s. Check configuration.", config.HttpServer.HTTPSKeyName, err)
 			errChannel <- err
 			return
 		}
-
 		// ListenAndServeTLS below takes filenames for the certificate and key but the raw data is coming from Vault, so must generate the tlsConfig from raw data first.
 		tlsConfig, err := webserver.generateTLSConfig([]byte(httpsCert), []byte(httpsKey))
 		if err != nil {
 			lc.Errorf("unable to generate a TLS configuration.", err)
 			errChannel <- err
 			return
 		}
-
 		svr.TLSConfig = tlsConfig
-
 		lc.Infof("Starting HTTPS Web Server on address %s", addr)
-
 		// ListenAndServeTLS takes filenames for the certificate and key but the raw data is coming from Vault
 		// empty strings will make the server use the certificate and key from tls.Config{}
 		errChannel <- svr.ListenAndServeTLS("", "")
```

Comment:
How does this work with the `ln` setup above?

Reply:
oh i'd missed this one. nice catch - thanks. i'll update